autoemulate/calibration/bayes.py

```diff
@@ -41,7 +43,15 @@ def __init__(  # noqa: PLR0913
         observations: dict[str, TensorLike]
             A dictionary of observations for each output.
         observation_noise: float |  dict[str, float]
-            A single value or a dictionary of values (one per output). Defaults to 0.1.
+            A single value or a dictionary of values (one per output) of measurement
+            noise measured in terms of standard deviation. Defaults to 0.1.
+        model_uncertainty: bool
+            Whether to include the variance associated with model predictions when
+            calculating the likelihood. Defaults to False.
+        model_discrepancy:
```

Comment:
if we are asking the user to specify model discrepancy as variance, then maybe we should specify observation noise as a variance too? Either way, it would be better if these are consistent with each other.

We should similarly specify what the expected unit is in the history matching dosctrings (currently it's unclear but in the variable names we treat everything is a variance).

Reply:
Yeah agree it would be good to have this consistency - it seems like treating all as variance might be most intuitive since we can add these up? I'll update the history matching docstring.

Reply:
Variance is updated in [4b313a1](https://github.com/alan-turing-institute/autoemulate/pull/765/commits/4b313a1afece2beb3eb1970549a37e71bf7e0838).